presets/4.3/tune/Tehllama_GTB339_3in_SWPR-Spec_2S-3S_Race_TUNE_Options.txt

```diff
@@ -0,0 +1,173 @@
+#$ TITLE: 3in Lightweight Race GTB339 3S/2S Tune
+#$ FIRMWARE_VERSION: 4.3
+#$ CATEGORY: TUNE
```

Comment:
i still don't agree this is a TUNE preset. I think this has to be in OTHER.
With the TUNE presets people are guaranteed not to have any left overs if they try different TUNE presets.

Your presets have the following left overs:
auto_profile_cell_count
changing profile
motor_poles
vbat_sag_compensation 

all this is not alloowed in the TUNE unless added in the UNCHECKED options.

OTHER category gives way more flexibility and people will understand they will have leftovers.
Also, it is good to mention these leftovers in the description.

I also think this PR is getting too complicated. Let's work on one preset at the time if they are that complex.
THanks!

Reply:
The auto-profile-select is somewhat integral to these, but I can definitely add the VbatSagComp into the options bin for each.
The motor pole forcing is only for specific spec racing builds (12P 1105 5500KV Mamba or 1404 TinyTurners)... I can wrap them in an option too.